recipe/meta.yaml

```diff
@@ -9,23 +9,24 @@ source:
         - CharScanner.patch
 
 build:
-    number: 0
+    number: 1
     skip: True  # [win]
+    skip: True  # [py3k]
 
 requirements:
     build:
-        - python >=2.7,<3
+        - python
     run:
         - python
 
 test:
     commands:
-        - antlr --help
+        - antlr-config --version
```

Comment:
... but any third party library that need to use `antlr` headers (`nco`) are OK.